src/main/java/bio/terra/stairway/FlightContext.java

```diff
@@ -194,4 +194,19 @@ public String toString() {
         .append("flightStatus", flightStatus)
         .toString();
   }
+
+  public String toStringTrunc() {
+    return new ToStringBuilder(this)
+        .append("stairway", stairway)
+        .append("flightId", flightId)
+        .append("flightClassName", flightClassName)
+        // .append("inputParameters", inputParameters)
+        .append("workingMap", workingMap)
```

Comment:
Do you think it would make sense to show input parameters up to a point?  Say, show the first 500 characters or something like that?  For cases where the input is pretty small, it still might be helpful to have this information.  Do the inputs get stored in the DB?  I just want to make sure that we don't lose information that may end up being critical to debug.